gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/CSVBase.java

```diff
@@ -95,24 +105,50 @@ protected String getEdgePath() {
     return csvRoot + EDGE_PATH;
   }
 
+  protected String getGraphHeadCSVPath() {
+    return csvRoot + GRAPH_HEAD_FILE;
+  }
+
+  protected String getVertexCSVPath() {
+    return csvRoot + VERTEX_FILE;
+  }
+
+  protected String getEdgeCSVPath() {
+    return csvRoot + EDGE_FILE;
+  }
+
   /**
    * Returns the path to the vertex file containing only vertices with the specified label.
    *
    * @param label vertex label
```

Comment:
It's the graphHead label not the vertex label. Same for description 2 lines above.